lib/widgets/screens/sign_up/sign_up_gender.dart

```diff
@@ -48,12 +61,12 @@ class _SignUpGenderState extends State<SignUpGender> {
                     ),
                   ),
                   trailing: Radio<String>(
-                    value: l10n
-                        .signupGenderOptionWoman, //TODO - use content provider textID
+                    value: l10n.signupGenderOptionWoman,
```

Comment:
Yes I was hesitant to use this approach, but wanted to match the options given in the UX design. I used the wording from UX and mapped to the corresponding textIds. If you plan on changing these options then definitively I can just put whatever is in translatedValue.

Reply:
Sounds good, in a future update we can change the default `translatedValue`s in the backend to match the latest values from the wireframes/PRD, and update this here. Maybe it could be batched with the business stage change

Reply:
I just pushed a commit to use the translatedValues:

![Screenshot 2023-09-29 at 9 35 14 AM](https://github.com/micromentor-team/mm-flutter-app/assets/25093428/178f37e1-a898-4d29-8880-6eb09586666d)